### PR TITLE
 chore: add tests to verify evaluated expressions

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

File: test/test-evaluated-expressions.ts [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+describe('debugger provides useful information', () => {
+  let api: debugapi.DebugApi;
+  const config =
+      extend({}, defaultConfig, {allowExpressions: true, forceNewAgent_: true});
+
+  before(done => {
+    // TODO: It appears `logLevel` is a typo and should be `level`.  However,
+    //       with this change, the tests fail.  Resolve this.
+    const logger =
+        new common.logger({levelLevel: config.logLevel} as {} as LoggerOptions);
```
Comment (kjin) [inline test/test-evaluated-expressions.ts]: `levelLevel` looks like a typo here?

File: test/test-evaluated-expressions.ts [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    scanner.scan(true, config.workingDirectory, /.js$/).then(fileStats => {
```
Comment (kjin) [inline test/test-evaluated-expressions.ts]: Nit: `/\.js$/` here and elsewhere

File: test/test-evaluated-expressions.ts (hunk 1 above)
Comment (DominicKramer) [reply test/test-evaluated-expressions.ts]: Fixed

File: test/test-evaluated-expressions.ts (hunk 2 above)
Comment (DominicKramer) [reply test/test-evaluated-expressions.ts]: Fixed

**Pull request merged**